src/main/java/sparksoniq/spark/closures/InitialForClauseClosure.java

```diff
@@ -32,7 +32,7 @@ public InitialForClauseClosure(String variableName) {
 
     @Override
     public FlworTuple call(Item v1) throws Exception {
-        FlworTuple result = new FlworTuple();
+        FlworTuple result = new FlworTuple(1);
```

Comment:
Maybe add secondary constructor with no args for the 1 case?

Reply:
Yes, I could do that -- however, I don't think it will ever happen that we do not know the size in advance. Actually, in the end, FLWOR tuples will be done with DataFrames because their structure is known in advance.